invoices/invoiceregistry_test.go

```diff
@@ -20,7 +19,7 @@ func TestSettleInvoice(t *testing.T) {
 	defer ctx.cleanup()
 
 	allSubscriptions, err := ctx.registry.SubscribeNotifications(0, 0)
-	assert.Nil(t, err)
+	require.Nil(t, err)
```

Comment:
what's the difference? :p

Reply:
One will fail the test, the other will log, then fail at the end.